### DISCUSSION TITLE
New search option

### DISCUSSION THREAD

**Discussion by BurkSchu** [sumatrapdfreader/sumatrapdf#4042] [Ideas]
I have one wish: That Sumatra also finds words that have been separated. xpdfsearch for Total Commander can do this. I initially searched in vain for the word I was looking for in xpdfsearch hits. After searching for components, I found it after all!

**Comment by GitHubRulesOK** [comment]
search in MuPDF is basic and SumatraPDF does not add much to that, there are differences depending on source document but it is also dependant on format so search through over 2000 pages takes 8 seconds before finding a hit (same as MuPDF, however MuPDF will show 3 entries but SumatraPDF only one until you click forward) 

Acrobat with its more complex matching takes 24 seconds.

So in this very rough test that's about 3 times as long and MuPDF & Acrobat have to be told to start but SumatraPDF starts looking with the first character then the next etc. so gets a head start.
